_posts/2022-01-10-running-puma-in-aws.md

```diff
@@ -141,6 +142,7 @@ containers:
         exec:
           command: ["sh", "-c", "sleep 40"]
 ```
+{% endraw %}
```

Comment:
Is there a reason we updated an old blog post in this PR? What does the `{% raw %}` do here?

Reply:
Fixes the following warning due to include tags that are being evaluated:
```
    Liquid Warning: Liquid syntax error (line 120): Expected end_of_string but found string in "{{ include "application.apps.name" . }}" in _posts/2022-01-10-running-puma-in-aws.md
    Liquid Warning: Liquid syntax error (line 121): Expected end_of_string but found string in "{{ include "container.image" . }}" in _posts/2022-01-10-running-puma-in-aws.md
```